CONTRIBUTING.md

```diff
@@ -5,4 +5,72 @@
 You are welcome to contribute to Dynatrace Operator.
 Use issues for discussing proposals or to raise a question.
 If you have improvements to Dynatrace Operator, please submit your pull request.
-For those just getting started, consult this  [guide](https://help.github.com/articles/creating-a-pull-request-from-a-fork/).
\ No newline at end of file
+For those just getting started, consult this  [guide](https://help.github.com/articles/creating-a-pull-request-from-a-fork/).
+
+## Coding style guide
+
+### General
+- Use descriptive names (`namespace` is better then `ns`, `dynakube` is better then `dk`, etc.;)
+- Avoid using `client.Client` for 'getting' resources, use `client.Reader` (also known as `apiReader`) instead.
+  - `client.Client` uses a cache (or tries to) that require more permissions then normally, and can also give you outdated results.
+
+### Logging
+
+#### Do's
+- Use I global `logger` per package, should be named `log` and be declared in the `config.go` of a package. (temporary solution)
+  - In case of larger packages its advisable to introduce separate loggers for different parts of the package, these sub-loggers should still be derived from the main logger of the package and given a name.
```

Comment:
```suggestion
  - In case of larger packages it's advisable to introduce separate loggers for different parts of the package, these sub-loggers should still be derived from the main logger of the package and given a name.
```